src/annotator/components/ContentInfoBanner.js

```diff
@@ -8,39 +15,108 @@ import { LabeledButton, Link } from '@hypothesis/frontend-shared';
  * A banner that displays information about the current document and the entity
  * that is providing access to it (eg. JSTOR).
  *
+ * layout columns:
+ *  - Logo
+ *  - Container title (only shown on screens at `2xl` breakpoint and wider)
+ *  - Item title with previous and next links
+ *
  * @param {object} props
  *   @param {ContentInfoConfig} props.info
- *   @param {() => void} props.onClose
  */
-export default function ContentInfoBanner({ info, onClose }) {
+export default function ContentInfoBanner({ info }) {
   return (
-    <div className="flex items-center border-b gap-x-4 px-2 py-1 bg-white text-annotator-lg">
-      <Link href={info.logo.link} target="_blank" data-testid="logo-link">
-        <img src={info.logo.logo} alt={info.logo.title} />
-      </Link>
-      <div className="grow">
-        {info.links.previousItem && (
-          <Link href={info.links.previousItem} target="_blank">
-            Previous article
+    <div
+      className={classnames(
+        'h-10 bg-white px-4 text-slate-7 text-annotator-base border-b',
+        'grid items-center',
+        // Two columns in narrower viewports; three in wider
+        'grid-cols-[100px_minmax(0,auto)] gap-x-4',
+        '2xl:grid-cols-[100px_minmax(0,auto)_minmax(0,auto)] 2xl:gap-x-3'
```

Comment:
On my 13" MacBook Pro, which has a CSS screen size of 1440x900px, I never see the three-column layout. According to [Tailwind's docs](https://tailwindcss.com/docs/screens#using-custom-screen-names), 2xl is 1536px.

Reply:
Would be interesting to see what this looks like with the longest article and journal titles we can find. I'm afraid I don't have an example to hand.

Reply:
Well, yes, because :facepalm: I didn't move over some breakpoint definitions from my working branch into the tailwind config. To extend the continuum of our current breakpoints, I intended to add:

```
        xl: '1024px',
        '2xl': '1220px',
        '3xl': '1400px',
```

So, `2xl` at `1220px`. This is, as you might observe, quite conservative still, but necessary because we have to account for the possibility of A) the sidebar being open and B) not being in side-by-side mode (sidebar obscures part of the content).

Reply:
> Would be interesting to see what this looks like with the longest article and journal titles we can find. I'm afraid I don't have an example to hand.

I did quite a bit of that when prototyping this design...